changelog.json

```diff
@@ -1,5 +1,22 @@
 {
   "releases": {
+    "1.7.0-beta0": [
+      "[New] Add Recent Group to Repo List - #7132",
+      "[Added] Explicit refresh of PR list and avoid skeleton loading screen when we have PRs to show - #7027",
+      "[Fixed] Cancel active selection before swapping document - #7131",
+      "[Fixed] Don't crash when failing to create log file - #7096",
+      "[Fixed] Ensure push goes to the right remote - #6986",
+      "[Fixed] Fix mistaken classification of all crashes as being related to launch - #7126",
+      "[Fixed] Make loadAndCacheUser more resilient to bad input - #6945",
+      "[Fixed] Only trigger conflicts flow for the selected repository - #7018",
+      "[Fixed] Prevent feedback loop toggling repository missing flag - #7062",
```

Comment:
This fix was included in the `1.6.4` hotfix https://github.com/desktop/desktop/pull/7063 and can be dropped from this list :fire::fire::fire: